README.md

```diff
@@ -19,3 +19,27 @@ quantitatively reason with them.
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+## Answer
+
+Choosing the first element gives us a 50% chance to pick a good pivot. In order for median-of-three to be better, the odds of picking a good pivot must be higher for this.
+
+In order for a selected pivot to be a good pivot, it must create two partitions of a least size $3n/4$ or $3/4$ the size of the original array.
+
+There are three ways for a pivot to be, good, bad, or amazing. This gives us three cases for each, where each case is equally likely. This gives us 3^3 possibilities, as there are three options and three slots, first, median, and last. Meaning there are 27 possible combinations of pivots. 
+We can find the odds of each case happening easily.
+The cases are as follows bad = B, good = G, and too good = T:
+(BBB) (BBG) (BBT) (BGB) (BTB) (BGG) (BGT) (BTT) (BTG)
+(GGG) (GGB) (GGT) (GBG) (GTG) (GBB) (GBT) (GTT) (GTB)
+(TTT) (TTG) (TTB) (TGT) (TBT) (TGG) (TGB) (TBB) (TBG)
+Looking at these, we can see that the median value is good in 3 of the top row cases, 3 of the middle row cases, and 3 of the bottom row cases.
```

Comment:
But not all of these cases have the same probability, as the middle half is larger than the first and last quarter.

Reply:
Redid all the calculations to reflect the good pivots being twice as likely as either version of the bad pivots.